ui/admin/app/templates/scopes/scope/aliases/index.hbs

```diff
@@ -18,155 +18,181 @@
-            </B.Td>
-            <B.Td>
-              {{#if B.data.alias.destination_id}}
-                <Hds::Copy::Snippet
-                  @textToCopy={{B.data.alias.destination_id}}
-                  @color='secondary'
-                />
-              {{/if}}
-            </B.Td>
-            <B.Td>
-              {{#let
-                (hash
-                  canUpdate=(can 'update model' B.data.alias)
-                  canDelete=(can 'delete model' B.data.alias)
-                )
-                as |perms|
-              }}
-                {{#if (or perms.canUpdate perms.canDelete)}}
-                  <Hds::Dropdown as |dd|>
+  <page.body class='search-filtering'>
+    {{#if @model.aliasesExist}}
+      <Hds::SegmentedGroup as |S|>
+        <S.TextInput
+          @value={{this.search}}
+          @type='search'
+          placeholder={{t 'actions.search'}}
+          aria-label={{t 'actions.search'}}
+          {{on 'input' this.handleSearchInput}}
+        />
+      </Hds::SegmentedGroup>
+
+      {{#if @model.aliases}}
+        <Hds::Table
+          @model={{@model.aliases}}
+          @columns={{array
+            (hash label=(t 'resources.alias.title'))
+            (hash label=(t 'resources.alias.titles.points_to'))
+            (hash label=(t 'resources.scope.title'))
+            (hash label=(t 'resources.alias.titles.resource_id'))
+            (hash label=(t 'titles.actions') isVisuallyHidden=true)
+          }}
+          @valign='middle'
+        >
+          <:body as |B|>
+            <B.Tr data-test-alias-row={{B.data.alias.id}}>
+              <B.Td>
+                {{#if (can 'read model' B.data.alias)}}
+                  <LinkTo
+                    @route='scopes.scope.aliases.alias'
+                    @model={{B.data.alias.id}}
+                  >
+                    {{B.data.alias.value}}
+                  </LinkTo>
+                {{else}}
+                  <Hds::Text::Body @tag='p'>
+                    {{B.data.alias.value}}
+                  </Hds::Text::Body>
+                {{/if}}
+                {{#if B.data.alias.description}}
+                  <Hds::Text::Body @tag='p'>
+                    {{B.data.alias.description}}
+                  </Hds::Text::Body>
+                {{/if}}
+              </B.Td>
+              <B.Td>
+                {{#if B.data.target.displayName}}
+                  <div class='aliases-list-cell-item'>
+                    <span><FlightIcon @name='target' /></span>
+                    <span> {{B.data.target.displayName}} </span>
+                  </div>
+                {{else}}
+                  <Hds::Text::Body
+                    @tag='p'
+                    @size='100'
+                    @weight='medium'
+                    @color='faint'
+                  >
+                    {{t 'titles.not-set'}}
+                  </Hds::Text::Body>
+                {{/if}}
+              </B.Td>
```

Comment:
I know this is a limitation of us not wanting to load all targets for the filter right now but without the filter, people are definitely going to think it's a bug that when they search the target name it doesn't filter down the table 😕

Reply:
For other resources that don't require a name, we display the resource ID. I know the ID isn't as helpful as a name but it could still be better than `Not Set`.

Reply:
I think `Not Set` is different as there could be an alias that's not pointing to any target yet so it's in the absence of the target. `displayName` would still display the ID if the target name doesn't exist

Reply:
hmm good thinking, perhaps an `Hds::TooltipButton` in the `Points to` table header letting user know they can only search on id and not name? I'll reach out to design on this.

Reply:
update: no changes are going to be made in regards to this, [slack thread ](https://hashicorp.slack.com/archives/C012GTH1C4E/p1727301278916459?thread_ts=1726778606.934579&cid=C012GTH1C4E) for context